sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -465,7 +496,17 @@ def plot(self, ax=None, n_cols=3, line_kw=None, contour_kw=None):
         default_contour_kws = {"alpha": 0.75}
         contour_kw = {**default_contour_kws, **contour_kw}
 
+        individual_line_kw = line_kw.copy()
+        if self.individual == 'both':
+            individual_line_kw['alpha'] = 0.3
+            individual_line_kw['linewidth'] = 0.5
```

Comment:
I think that we should force each line to have the same color by default (the change of color does not have any meaning indeed).

Reply:
Fixed in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/8f7be27145e597c8171a681eb94cd565626f5768